x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/components/document_fields/field_parameters/analyzers_parameter.tsx

```diff
@@ -26,6 +27,12 @@ export const AnalyzersParameter = ({ field, withSearchQuoteAnalyzer = false }: P
       title={i18n.translate('xpack.idxMgmt.mappingsEditor.analyzersSectionTitle', {
         defaultMessage: 'Analyzers',
       })}
+      docLink={{
```

Comment:
This currently links to https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-analyzers.html

<img width="716" alt="Screen Shot 2019-12-16 at 3 48 12 PM" src="https://user-images.githubusercontent.com/5226211/70954022-7d8f7600-2039-11ea-91ec-d74de9a48b31.png">

WDYT? There is also documentation for `analyzer` (which includes `search_quote_analyzer`) and `search_analyzer`.  I'm wondering if it would be better to link to these under each field instead, although it may get a little tricky when "Use the same analyzers..." is checked.

Reply:
I think we should link to https://www.elastic.co/guide/en/elasticsearch/reference/current/analyzer.html. This is more consistent with the links for the other parameters and it also contains a link to `analysis-analyzers.html` so users will still have access to that information.